### DISCUSSION TITLE
Convert wire containg 4 bits to 1 bit.

### DISCUSSION THREAD

**Discussion by D-Mielewczyk** [hneemann/Digital#964] [Q&A]
I have a wire containing **4 bits** of information. I would like to only know if the number is 0 or greater and store it in **1-bit** information.  What is the optimal way of doing that?

**Comment by TheWildJarvi** [comment]
Use a bus splitter, or the bits together.

**Reply by D-Mielewczyk** [reply]
By bus splitter do you mean bit selector? I don't see a bus splitter option in the program. My approach:
![image](https://user-images.githubusercontent.com/55713797/172436130-a6224395-3f2d-48e7-bd5f-d917160d47d1.png)

**Reply by TheWildJarvi** [reply]
Components>wires>splitter/merger

**Reply by hneemann** [reply]
See here:

[splitter.zip](https://github.com/hneemann/Digital/files/8855260/splitter.zip)
